api/app/signals/apps/signals/templates/signals/swagger/openapi.yaml

```diff
@@ -0,0 +1,399 @@
+openapi: 3.0.0
+info:
+  title: Signalen Informatievoorziening Amsterdam (SIA) API spec V1"
+  description: One of the tasks of the municipality of Amsterdam is to maintain
+    public spaces like roads, parks. When citizens have complaints about public
+    spaces they can leave these complaints with the municipality. SIA, the
+    Signalen Informatievoorziening Amsterdam, receives these complaints and is
+    used to track progress on their resolution. SIA provides an API that is used
+    both by the SIA frontend and external systems that integrate with SIA.
+  contact:
+    email: "datapunt@amsterdam.nl"
+  version: "TBD"
+servers:
+  - url: {{global_api_root}}
+    description: "Datapunt acceptatie server for SIA testing and acceptation."
+paths:
+  /signals/v1/private/signals/:
+    get:
+      description: Signals list endpoint
+      parameters:
+        - name: "created_before"
+          in: query
+          description: Filter returns only signals created before specified
+            datetime.
+          schema:
+            type: string
+            format: date-time
```

Comment:
We still need to add an example for date-time, see previous comment for the format.